src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -132,4 +141,107 @@ public CommandBase turnDegrees(double degrees, double speed) {
         cmd.setName("Turn Degrees");
         return cmd;
     }
+
+    public CommandBase slowTurn(boolean direction) {
+        CommandBase cmd = new RunCommand(() -> {
+            if (direction == true) {
```

Comment:
I don't like the use of "direction" as a Boolean here. Either make it the speed itself, or change the name to turning left/turningRight.

Also, you can just say "if (direction)" and "else" in this block - the existing code is very redundant